config/locales/en/views/searches.yml

```diff
@@ -41,7 +41,6 @@ en:
         providers: Providers attending
       offence:
         caption: Offence list
-        date: Date
```

Comment:
just to check, was this removed on purpose?

Reply:
Yes - offences_v2 had an extra column showing the _date_ of the offence, which isn't available in the v1 API. (Side note: we should ticket up _adding_ the date of the offence to the v1 payload so that we can add this column to the VCD UI without having to switch to v2)